pkg/controller/factory/factory.go

```diff
@@ -91,6 +126,16 @@ func (f *Factory) WithSyncContext(ctx SyncContext) *Factory {
 	return f
 }
 
+// DefaultConstQueueKey returns a "key" const that is used to trigger a string based controllers.
+func DefaultConstQueueKey() string {
+	return defaultConstQueueKey(nil)
+}
+
+// defaultConstQueueKey is a wrapper for string const queue key function.
+func defaultConstQueueKey(runtime.Object) string {
```

Comment:
why do we need this and the one above?

Reply:
Would make `defaultConstQueueKey` available as `DefaultConstKeyFunc` and `defaultStringQueueKey` available as `DefaultConstQueueKey`.

Reply:
no, keep it simple, just make one public const and controllers can reference